### ISSUE TITLE
Works on FA506QM.308 with let_it_burn=1

### ISSUE THREAD

**Issue by bonohub13** [hackbnw/faustus#106]
Works great on FA506QM.308 using let_it_burn=1, however could not get it to work with OpenRGB (keyboard LED was detected, but could not configure anything).

---

## Required Info
OS: Archlinux

Output of `uname -a`:
```
Linux gorone-chan 5.19.0-xanmod2-2-zen3 #1 SMP Mon, 08 Aug 2022 09:52:43 +0000 x86_64 GNU/Linux
```

`dmidecode` info:
```
BIOS Information
        Vendor: American Megatrends International, LLC.
        Version: FA506QM.308
        Release Date: 12/10/2021
--
Base Board Information
        Manufacturer: ASUSTeK COMPUTER INC.
        Product Name: FA506QM
        Version: 1.0
```

DSDT and everything above:
[FA506QM_feedback.tar.gz](https://github.com/hackbnw/faustus/files/9313170/FA506QM_feedback.tar.gz)

---

Really appreciate the work you've put in! It was like a Christmas miracle when the keyboard LED started to work! Thanks

---
EDIT:
When running `./set_rgb.sh`, the following stuff was output.
```
./set_rgb.sh: line 19: /sys/devices/platform/faustus/kbbl/kbbl_set_auramode: Permission denied
./set_rgb.sh: line 21: /sys/devices/platform/faustus/kbbl/kbbl_set_auraspeed: Permission denied
```

Maybe something to do with OpenRGB not working properly? (dunno)

**Comment by bonohub13** [comment]
UPDATE:
Updated the bios to version 311 and confirmed it still works

To use the faustus driver working properly after reboot, I had to add the following options under modprobe.d and modules-load.d

/etc/modprobe.d/faustus.conf:
```
blacklist asus_wmi
blacklist asus_nb_wmi

options faustus let_it_burn=1
```

/etc/modules-load.d/faustus.conf
```
faustus
```

After adding these files, update the initramfs. (With Archlinux, run `sudo mkinitcpio -P`)

**Comment by bonohub13** [comment]
Apologies:
For the past month I was not using the faustus driver from this repository.
Instead, I was using the [fork](https://github.com/Ann1kaB/faustus) when I installed it from the AUR (Arch User Repository). Sorry for the confusion.
I tried the faustus driver from this repository, but could not get the aura button to work to change the color for the keyboard LEDs. I could get it to change with tuf-manager.
I am hoping if the patch that Ann1kaB applied to his/her fork that made the aura button to work could be merged upstream.
